cmd/immuadmin/command/database.go

```diff
@@ -37,39 +37,49 @@ func addDbUpdateFlags(c *cobra.Command) {
 	c.Flags().String("replication-follower-username", "", "set username used for replication")
 	c.Flags().String("replication-follower-password", "", "set password used for replication")
 	c.Flags().Uint32("write-tx-header-version", 1, "set write tx header version (use 0 for compatibility with immudb 1.1, 1 for immudb 1.2+)")
+	c.Flags().Bool("autoload", true, "enable database autoloading")
 }
 
 func (cl *commandline) database(cmd *cobra.Command) {
-	ccmd := &cobra.Command{
-		Use:     "database",
-		Short:   "Issue all database commands",
-		Aliases: []string{"d"},
-		//PersistentPreRunE: cl.ConfigChain(cl.connect),
+	dbCmd := &cobra.Command{
+		Use:               "database",
+		Short:             "Issue all database commands",
+		Aliases:           []string{"d"},
 		PersistentPostRun: cl.disconnect,
-		ValidArgs:         []string{"list", "create", "update", "use", "clean"},
```

Comment:
We're changing clean to compact - maybe we should support both action names for backward compatibility?

There could be some admin scripts running compaction periodically already out there.

Reply:
actually the command was changes couple of releases ago, this was a left over...